content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+# Multiple Active Runs in One Process (Reinit Behavior)
+
+**Quick Reference**  
+- **`return_previous`**: Return the existing run; no new run is created.  
```

Comment:
If there are already multiple runs, this returns the most recently created one that isn't finished.

See https://github.com/wandb/wandb/blob/0d5b23a564812dfd159edf3210d5a0aeda85df08/wandb/sdk/wandb_settings.py#L374-L399